content/applications/general/in_app_purchase.rst

```diff
@@ -1,59 +1,161 @@
+   - :doc:`Enrich your contacts base with Partner Autocomplete
+     </applications/sales/crm/optimize/partner_autocomplete>`
+   - :doc:`SMS essentials </applications/marketing/sms_marketing/essentials/sms_essentials>`
+
+.. _in_app_purchase/credits:
+
+IAP credits
+===========
+
+Every time an IAP service is used, the prepaid credits for that service are spent. Odoo prompts the
+purchase of more credits when there are not enough credits to continue using the service. Email
+alerts can also be set up for when :ref:`credits are low <in_app_purchase/low-credits>`.
+
+Credits are purchased in "packs" from the `Odoo IAP website
+<https://iap.odoo.com/iap/all-in-app-services>`_, and pricing is specific to each service.
+
+.. example::
+   The `SMS service <https://iap.odoo.com/iap/in-app-services/1>`_ has four packs available, in
+   denominations of:
+
+   - **10** credits
+   - **100** credits
+   - **500** credits
+   - **1,000** credits
+
+   The number of credits consumed depends on the length of the SMS and the country of destination.
+   For more information, refer to the :doc:`SMS Pricing and FAQ
+   </applications/marketing/sms_marketing/pricing/pricing_and_faq>` documentation.
+
+.. _iap/buying_credits:
+
+Buy credits
+-----------
+
+If there are not enough credits to perform a task, the database automatically prompts the purchase
+of more credits. Users can check the current balance of credits for each service, and manually
+purchase more credits through the service's :guilabel:`Account Information` page.
+
+Manually buy credits
+~~~~~~~~~~~~~~~~~~~~
+
+   .. image:: in_app_purchase/view-services.png
+      :align: center
+      :alt: The Settings app showing the Odoo IAP heading and View My Services button.
+
+#. Go to the *Settings* application.
+#. Type `IAP` in the search bar at the top.
+#. Click :guilabel:`View My Services`.
+
+   .. image:: in_app_purchase/account-info.png
+      :align: center
+      :alt: The Account Information page for an IAP service showing the Buy Credit button.
+
+#. The available IAP accounts appear in a list view. Click on the desired IAP account to view the
+   :guilabel:`Account Information` page.
+#. On the :guilabel:`Account Information` page, click the :guilabel:`Buy Credit` button. A
+   :guilabel: `Buy Credits for (IAP Account)` page loads in a new tab.
+
+   .. image:: in_app_purchase/buy-pack.png
+      :align: center
+      :alt: The SMS service page on IAP.Odoo.com with four packs of credits available for purchase.
+
+#. Click :guilabel:`Buy` on the desired pack of credits, then follow the prompts to enter payment
+   details and confirm the order.
+#. Once the transaction is complete, the credits are available for use in the database.
+
+.. _in_app_purchase/low-credits:
+
+Low-credit notification
+~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to be notified when credits are low in order to avoid running out of credits while
+performing an In-App Purchases (IAP).
```

Comment:
```suggestion
It is possible to be notified when credits are low, in order to avoid running out of credits, while
using an IAP service. To do that, follow this process:
```